net/eoip/patches/100-add_LF_to_pidfile.patch

```diff
@@ -0,0 +1,11 @@
+--- a/eoip.c.orig	2013-09-25 22:50:48.272763057 +0200
```

Comment:
Is the source file to be patched really named `eoip.c.orig`  ?
I would assume that this should be `eoip.c` 

(Looking at your upper upstream source code archive, there is no file  `eoip.c.orig` )

Reply:
https://github.com/agustim/openwrt-linux-eoip/commit/2e433dd9156c672d7128f95ce624781485cb44d7